bundles/org.openhab.binding.omnilink/README.md

```diff
@@ -309,13 +309,10 @@ DateTime   OmniProTime   "Last Time Update [%1$ta %1$tR]"   <time>   {channel="o
 =Unknown
 ```
 
-### Example `omnilink.rules`
+### Example EMCAScript rule
 
-```
-rule "Update OmniPro Time"
-when
-  Time cron "0 0 0/1 1/1 * ? *"
-then
-  OmniProTime.sendCommand( new DateTimeType() )
-end
+``` javascript
+var omnilinkAction = actions.get("omnilink", "omnilink:controller:home");
+
+omnilinkAction.setDateTime("America/Denver");
```

Comment:
I believe I read somewhere that rule examples still need to be documented as DSL rules for consistency across all bindings. And **only** DSL rules to avoid redundancy.

Reply:
Hmm, well I would prefer to use JS because it is the direction OH3 is headed. It also makes it easier to implement in a new system because JS is so much more flexible. I haven't seen any formal declaration like that in the documentation any other thoughts from the other @openhab/add-ons-maintainers?

Reply:
I'm not aware that we standardize on DSL rules in docs, but i agree that while out DSL is not going away anytime soon, it is seeing a decline in usage in favor of our JS automation engine, I'm ok with starting to see more docs reference JS rules, but welcome others point of view.

Reply:
Supporting multiple scripting languages using tabs might also be an option. 

Like this:

https://www.openhab.org/docs/configuration/jsr223.html#example-rules-for-a-first-impression
https://github.com/openhab/openhab-docs/blob/main/configuration/jsr223.md#example-rules-for-a-first-impression

But I don't know if that also works here. :slightly_smiling_face:

Reply:
> But I don't know if that also works here. 🙂

I think that it does, I like that idea.